src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -294,22 +296,32 @@ public override void BuildFinishedHandler(object sender, BuildFinishedEventArgs
                 }
 
                 // Emit text like:
+                //   Framewook: ------
                 //     1 Warning(s)
                 //     0 Error(s)
                 // Don't color the line if it's zero. (Per Whidbey behavior.)
-                if (warningCount > 0)
+                // separate counts by framework
+                foreach (var counts in target_framwork_errorwarning)
                 {
-                    setColor(ConsoleColor.Yellow);
-                }
-                WriteLinePrettyFromResource(2, "WarningCount", warningCount);
-                resetColor();
+                    if (counts.Key == null || counts.Key.Length <= 0) continue;
 
-                if (errorCount > 0)
-                {
-                    setColor(ConsoleColor.Red);
+                    Console.WriteLine("Target Framwork: " + counts.Key);
+                    resetColor();
```

Comment:
Do you have to reset the color this many times? If you set it explicitly, I imagine that makes this call redundant, and you reset it at the end anyway.